package.json

```diff
@@ -24,9 +24,24 @@
     "dash",
     "MPEG-DASH"
   ],
-  "devDependencies": {
+  "browserify": {
+    "transform": [
+      "browserify-shim"
+    ]
+  },
+  "browserify-shim": {
+    "dashjs": "global:dashjs",
+    "video.js": "global:videojs"
+  },
+  "dependencies": {
     "dashjs": "^2.0.0",
+    "global": "^4.3.0",
+    "video.js": "^5.0.0"
+  },
+  "devDependencies": {
+    "browserify-shim": "^3.8.12",
```

Comment:
this would need to be a regular dependency if we want to allow users to require this module. Either that or the `browserify-shim` option should only be specified in the grunt config for the dist output.